README-PLAYBOOK.md

```diff
@@ -95,7 +95,8 @@ The top-level structure of the file is described as follows:
 | credentials | [Credentials](#credentials) object | ***Required*** | ***Required*** | ***Required*** | A [Credential](#credentials) object that defines the credentials used to authenticate to the selected provider `platform`.                                                                                                                                                                |
 | platform    | string                             | ***Required*** | ***Required*** | ***Required*** | For TLS Protect Datacenter, either `tpp` or `tlspdc`.<br/>For TLS Protect Cloud, either `vaas` or `tlspc`.<br/>For Firefly, use `firefly`.                                                                                                                                                |
 | trustBundle | string                             | *Optional*     | n/a            | *Optional*     | Used when [Connection.platform](#connection) is `tlspdc` or `firefly`.<br/>Defines path to PEM-formatted trust bundle that contains the root (and optionally intermediate certificates) to use to trust the TLS connection. If omitted, will attempt to use operating system trusted CAs. |
-| url         | string                             | ***Required*** | *Optional*     | ***Required*** | URL of the Venafi platform to connect to.<br/>If url string does not include `https://`, it will be added automatically.<br/>For connection to TLS Protect Datacenter, `url` must include the full API path (for example `https://tpp.company.com/vedsdk/`.                               |
+| url         | string                             | ***Required*** | *Optional*     | ***Required*** | URL of the Venafi platform to connect to.<br/>If url string does not include `https://`, it will be added automatically.<br/>For connection to TLS Protect Datacenter, `url` must include the full API path (for example `https://tpp.company.com/vedsdk/` <br/> For TLS Protect Cloud use `https://api.venafi.cloud` (US region, default) or `https://api.api.venafi.eu` (EU region).                             
```

Comment:
>For TLS Protect Cloud use `https://api.venafi.cloud` (US region, default) or `https://api.api.venafi.eu` (EU region). 
       
Could you change it to:

For TLS Protect Cloud you can specify the url using this parameter `https://api.venafi.cloud` (US region) or `https://api.api.venafi.eu` (EU region). If not set, will default to `https://api.venafi.cloud` (US region).

Reply:
The EU endpoint should be `https://api.venafi.eu` not `https://api.api.venafi.eu`

Reply:
Thank you for noticing, my brain auto corrected that one haha